bigquery_etl/public_data/publish_gcs_metadata.py

```diff
@@ -202,15 +203,7 @@ def main():
                 args.endpoint,
                 target,
             )
-
-            output_file = f"gs://{args.target_bucket}/all-datasets.json"
-            publish_all_datasets_metadata(gcs_table_metadata, output_file)
```

Comment:
Iterating through all project, `gcs_table_metadata` was empty in some cases overwriting `all-datasets.json` which resulted in an empty file.